akka-projection-grpc-tests/src/test/scala/akka/projection/grpc/replication/ReplicationSettingsSpec.scala

```diff
@@ -32,24 +32,36 @@ class ReplicationSettingsSpec extends AnyWordSpec with Matchers {
            # which of the replicas this node belongs to, should be the same
            # across the nodes of each replica Akka cluster.
            self-replica-id = dca
+
            # Pick it up from an environment variable to re-use the same config
            # without changes across replicas
            self-replica-id = ${?SELF_REPLICA}
+
            # max number of parallel in-flight (sent over sharding) entity updates
            # per consumer/projection
            parallel-updates = 8
+
            # Fail the replication stream (and restart with backoff) if completing
            # the write of a replicated event reaching the cluster takes more time
            # than this.
            entity-event-replication-timeout = 10s
+
+           # When enabled all events will be transferred from all replicas, otherwise
+           # only events from the origin replica will be transferred from the origin
+           # replica. When each replica is connected to each other replica it's most
+           # efficient to disable indirect replication.
+           indirect-replication = on
```

Comment:
default value is `off`, this is included in docs and maybe we should show `off`
however, then the test would be testing this

Reply:
yes